AK/StringFloatingPointConversions.cpp

```diff
@@ -0,0 +1,2278 @@
+/*
+ * Copyright (c) 2022, Dan Klishch <danilklishch@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <AK/BitCast.h>
+#include <AK/BuiltinWrappers.h>
+#include <AK/StringFloatingPointConversions.h>
+
+namespace AK {
+
+// This entire algorithm is an implementation of the paper: Ryu: Fast Float-to-String Conversion
+// by Ulf Adams, available at https://dl.acm.org/doi/pdf/10.1145/3192366.3192369 and an implemenetation
+// at https://github.com/ulfjack/ryu.
```

Comment:
This link doesn't work because of the period, maybe we can put a space between the link and period.